www/src/slices/mapper/slice.ts

```diff
@@ -132,3 +132,7 @@ let slice = createSlice({
 
 export let mapperReducer = slice.reducer
 export let mapperActions = slice.actions
+
+export function isCurrentZoomDefault(zoom: number) {
```

Comment:
I wonder if it's more in line with react to make this part of the state. E.g. on line 18 we have a "zoomedEnoughToShowPolygons" state so we could have some kind of "panToFramework" state?